apps/src/studio/studio.js

```diff
@@ -3738,10 +3754,6 @@ Studio.onPuzzleComplete = function() {
     Studio.result = ResultType.SUCCESS;
   }
 
-  // Stop everything on screen
```

Comment:
We're reconfiguring this call instead of writing a new function where this part is taken out because we're reworking how freeplay levels work for all of studio, correct?

Reply:
if I understand the question correctly, yes. I wanted a basic `onPuzzleComplete` call to act exactly the same as it used to, but be able to call the rest of the function (minus these 2 deleted lines and with the `NoFeedback` version of `onComplete`) from inside the Run button handler.